internal/jobs/pubkey_upload_aws_job.go

```diff
@@ -5,8 +5,10 @@ import (
 	"errors"
 	"fmt"
 
-	"github.com/RHEnVision/provisioning-backend/internal/clients/ec2"
-	"github.com/RHEnVision/provisioning-backend/internal/clients/sts"
+	_ "github.com/RHEnVision/provisioning-backend/internal/clients/http/sts"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/clients/http/ec2"
```

Comment:
we need to pull the error to the clients package, so we do not need to import this ideally :) good to do in follow-up.